src/app/meds-n-labels/activity-1/activity-1.component.ts

```diff
@@ -23,26 +24,24 @@ export class MatchmedsComponent implements OnInit {
     canvasTop: number;
     canvasLeft: number;
 
-    height = 320;
-    width = 75;
+    HEIGHT = 600;
```

Comment:
why capitalized.. am I missing something here.. caps are supposed to be just for constants right ?